### ISSUE TITLE
The app refuses to save more than one "proton-session" cookies records set

### ISSUE THREAD

**Issue by stefwill** [vladimiry/ElectronMail#672]
I have multiple proton accounts, and everything has been going swimmingly until recently.
Now I get this error message alert box filling my screen with duplicate error messages that I need to close to be able to read my mail.
`The app refuses to save more than one "proton-session" cookies records set (accessTokens count: 2, refreshTokens count: 2).`

I have been unable to find anything related to this issue.
Not sure what other info to provide.
Cheers

**Comment by vladimiry** [comment]
The situation like this not supposed to happen, so the app detects invalid case and shows the error message you posted. Regarding preventing this situation to happen, clearly there is a session saving/applying-related edge case that the app didn't take into the account yet.

The workaround is removing `session.bin` located in the [settings folder](https://github.com/vladimiry/ElectronMail/blob/master/images/open-settings-folder.jpg) and restarting the app. This will force you to sign in into the email accounts due to the saved sessions data got dropped.

**Comment by stefwill** [comment]
Thank you.
That has done the trick.

**Comment by vladimiry** [comment]
In the next release, the app is going to save multiple "proton-session" cookies records. The proper approach would be in keeping only the actual/valid records set, but this requires thorough debugging, which at this time I'm going to replace with filtering by Proton as I tested this change for some time and the actual/valid record is being well picked by Proton since I get successfully signed in.

To recap, the annoying above-described workaround should not be needed anymore since the next app release (the change is already available in the `wip` branch).
